holoviews/core/data/__init__.py

```diff
@@ -274,22 +274,25 @@ def sort(self, by=[], reverse=False):
         return self.clone(sorted_columns)
 
 
-    def range(self, dim, data_range=True):
+    def range(self, dim, data_range=True, dimension_range=True):
```

Comment:
Would be good to update the docstring to explain this new flag.

Reply:
Agreed, should be done here and in the Dimensioned baseclass at least.